service/http/http.go

```diff
@@ -0,0 +1,275 @@
+// Package http implements the HTTP service for Flow.
+package http
```

Comment:
Is adding another top level directory the right call here? Would it make more sense to go under pkg/flow? Do we have guidelines on what makes something important enough for a tld?

Reply:
I'm gradually moving away from the pkg/ folder, most Flow things are top-level now (like component). 

I can move if it you want, though.

Reply:
I'm ok with that.